site/src/docs-components/color/ZenColorGroup.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from "react"
+import ZenColor from "./ZenColor"
+
+const colorGroupStyles: React.CSSProperties = {
+  display: "grid",
+  gridTemplateColumns: "1fr 1fr 1fr",
+  gridColumnGap: "2%",
+  justifyContent: "space-between",
+}
+
+const ZenColorGroup = ({ colors }) => (
+  <div style={colorGroupStyles}>
+    {Object.keys(colors)
+      .reverse()
+      .map(c => (
+        <ZenColor name={c} value={colors[c]} />
```

Comment:
Is this complaining about a missing `key` prop?

Reply:
Oh, weird... it's not, no error in the console on the branch preview nor in the editor. 🤔 Maybe the `name` prop is weird because of the HTML attribute `name` and that makes it unique or something?

Reply:
Anyway I'll add it!